pkgs/games/path-of-building/default.nix

```diff
@@ -59,6 +85,31 @@ stdenv.mkDerivation {
     )
   '';
 
+  desktopItems = [
+    (makeDesktopItem {
+      name = "Path of Building";
+      desktopName = "Path of Building";
+      comment = "Offline build planner for Path of Exile";
+      exec = "pobfrontend %U";
+      terminal = false;
+      type = "Application";
+      icon = fetchurl {
+        name = "PathOfBuilding-logo.png";
+        url = "https://avatars.githubusercontent.com/u/60926656?s=200&v=4";
```

Comment:
That's definitely not going to be stable long term.

Reply:
Also, icon should be an icon name ad not a path, so that icon themes work on it.

Reply:
I was unable to find an icon within the pob repo (I think it's embedded within the .exe), the closest I managed to find was the favicon within the repo for the website: https://raw.githubusercontent.com/PathOfBuildingCommunity/pathofbuildingcommunity.github.io/master/favicon.ico

Reply:
You can use imagemagick's convert to convert it to png.

Reply:
Ok, I've modified it to extract the icons from the exe file directly, no longer need to download the icon from somewhere else. :)